### PR TITLE
additional minor fixes when setting up server.

### PR DESCRIPTION
**Pull request by molleweide**
- add setup-util-btrfs
- check if btrfs exists in helper 
- check if linux in setup server or exit
- fix minor typo/issue in `secret` command

### CODE REVIEW

**Review by balupton — CHANGES_REQUESTED** [review]
Change this line to check for btrfs instead of curl, and uncomment

File: commands/setup-util-btrfs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+function setup_util_btrfs() (
+	source "$DOROTHY/sources/bash.bash"
+
+	# # improve performance
+	# if is-needle --quiet "$@" && ! is-needle --upgrade "$@" && ! is-needle --uninstall "$@" && command-exists curl; then
+	# 	return 0
```
Comment (balupton) [inline commands/setup-util-btrfs:8]: This one

**Comment by molleweide** [comment]
i force pushed the change but now I cant see the change here on github

**Comment by balupton** [comment]
merged in #178

**Pull request closed without merging**